app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+# from app.errors import * bad practice or something?
```

Comment:
Maybe you should delete this comment

Reply:
No it was a question for mentors, I wanted to add all from errors this way but flake8 didn't agree with me.

Reply:
Thank you!